lib/checkout.rb

```diff
@@ -0,0 +1,27 @@
+require 'dotenv/load'
+require 'twilio-ruby'
+
+class Checkout
+  # possibly unfortunate name for the class, but I would have expected it to also handle
+  # the payment and not just send an SMS
+
+  def initialize
+    @time = Time.now + 3600
+  end
+
+  def send_notification
+    account_sid = ENV['SID'] # Your Test Account SID from www.twilio.com/console/settings

```

Comment:
Nice use of ENV variables :) to make it even better, those values could be passed when initialising the `Checkout` instance